sensitivity.tex

```diff
@@ -53,24 +53,35 @@ \subsection{Breeding Ratio}
     \caption{Increasing the breeding ratio decreases the number of total \gls{UOX} \gls{UNF}
              required. The \glspl{ASTRID} produce more plutonium, reducing the plutonium influx
              from reprocessed \gls{UOX}.}
-    \label{fig:eu_tail}
+    \label{fig:br_uox}
 \end{figure}
 
 
 \subsection{Lifetime Extension of French \glspl{LWR}}
 Extending the lifetime of French \glspl{LWR} dramatically lowers the average
 monthly \gls{UOX} reprocessing demand, since the \gls{ASTRID} deployment becomes less
 aggressive. The plutonium demand is delayed allowing the reprocessing plant more
-time to produce plutonium for \gls{ASTRID} reactors. The
+time to prepare plutonium for \gls{ASTRID} reactors. The
 extended deployment schemes are illustrated in figure \ref{fig:ext}. 
 
+Figure \ref{fig:ext_uox} shows the decrease in the average monthly
+\gls{UOX} reprocessing burden with increased lifetime of \glspl{LWR},
+which reduces to the current capacity of the La Hague site if all the
+French \glspl{LWR} extended their operation for 20 years.
+Figure \ref{fig:ext_all} shows that lifetime extension has little
+effect on the average total monthly reprocessing demand, because
+the amount of plutonium in the \gls{ASTRID} used fuel remains the same.
+ The initial increase is due to the decrease in the time for \gls{MOX} 
+ reprocessing, since \gls{MOX} reprocessing is delayed with the
```

Comment:
It's not clear what you mean by "the time for MOX reprocessing" .

Reply:
What I'm trying to say is:
because average is `total/time`,
and since delaying ASTRID deployment (through LWR lifetime extensions),
delays the time MOX reprocessing starts (since the 'first' MOX UNF discharge is later), 
the `time for MOX reprocessing` decreases, leading to an increase in the average.
I wasn't sure how many steps I could jump..